athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/handlers/GlueMetadataHandler.java

```diff
@@ -449,4 +462,24 @@ protected static String getSourceTableName(Schema schema)
         }
         return ImmutableMap.of();
     }
+
+    protected static Map<String, String> getDateTimeFormatMapping(Table table)
```

Comment:
why protected? isnt this info available on the Schema object we return from getTable?

Reply:
also, if we do keep this, it needs javadoc

Reply:
Right now, the customer provides a mapping of column names (ex col1=Col1 where col1 is glue name and Col1 is ddb name). I thought it was better for the customer to do the same for date format mapping (i.e. col1=someformat instead of Col1=someformat) since col1 is glue name and the mapping is on glue table metadata.

this *could be* available on the schema object as is, but Ive added excludeKey in addMetadata is to prevent the un-normalized mapping to be added to metadata when we go through them. If i keep the raw map in the metadata, it would be on the schema object but it will have glue column name, not the normalized ddb column name.